lib/base/strings.dart

```diff
@@ -30,6 +30,11 @@ abstract final class Strings {
   static const loginForgotYourPasscode = 'Forgot your passcode?';
   static const loginForgot = 'Forgot?';
 
+  static const loginResendVerificationEmail = 'Resend verification email';
+  static const loginVerificationEmailSent = 'Verification email sent';
+  static String loginVerificationEMailBody(String email) =>
```

Comment:
```suggestion
  static String loginVerificationEmailBody(String email) =>
```
Of course this also applies anywhere it's used